db/config.cc

```diff
@@ -1470,6 +1470,14 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , alternator_max_expression_cache_entries_per_shard(this, "alternator_max_expression_cache_entries_per_shard", liveness::LiveUpdate, value_status::Used, 2000, "Maximum number of cached parsed request expressions, per shard.")
     , alternator_max_users_query_size_in_trace_output(this, "alternator_max_users_query_size_in_trace_output", liveness::LiveUpdate, value_status::Used, uint64_t(4096),
             "Maximum size of user's command in trace output (`alternator_op` entry). Larger traces will be truncated and have `<truncated>` message appended - which doesn't count to the maximum limit.")
+    , alternator_response_gzip_compression_level(this, "alternator_response_gzip_compression_level", liveness::LiveUpdate, value_status::Used, int8_t(-1),
+            "Controls gzip and deflate compression level for Alternator response bodies (if the client requests it via Accept-Encoding header).\n"
+            "Valid values:\n"
+            "\t-1 : Auto compression level (default)\n"
+            "\t0 : No compression (disables gzip/deflate)\n"
+            "\t1-9: Compression levels (1 = fastest, 9 = best compression)")
+    , alternator_response_gzip_compression_threshold_in_bytes(this, "alternator_response_gzip_compression_threshold_in_bytes", liveness::LiveUpdate, value_status::Used, uint64_t(4096),
```

Comment:
If this configuration option now is not specific to gzip and is the same for both supported compression algorithms, can we remove the word "gzip" from it's name?
Or maybe you didn't intend it to be for all compression algorithms we'll support in the future - just for gzip and defalte and not others? If so, why?